contributing.md

```diff
@@ -247,7 +247,7 @@ Example: `Fix typos in Foo scaladoc`
         committers to set. JIRA tends to unfortunately conflate "size" and "importance" in its 
         Priority field values. Their meaning is roughly:
              1. Blocker: pointless to release without this change as the release would be unusable 
-             to a large minority of users
+             to a large minority of users. Correctness and data loss issues should be considered Blockers.
```

Comment:
Since this is `contributing.md`, the process looks like the followings.
1. Contributor can make a JIRA issue with `Major` with `correctness` label.
2. Committers should validate this and upgrade it from `Major` to `Blocker`.

It's a gray area during the period between (1) and (2). Or, are we widely open for contributors to create `Blocker` issue with `correctness` label?

Reply:
I don't mind anyone setting Blocker if they really think it's a correctness or data-loss issue. I'll clarify that.